internal/workermanager/in_situ_worker.go

```diff
@@ -61,6 +65,10 @@ func (m *InSituWorker) EnqueueSyncAssetJob(ctx context.Context, service string)
 
 		for _, ast := range assets {
 			if err := m.discoveryRepo.Upsert(ctx, ast); err != nil {
+				if strings.Contains(err.Error(), "illegal_argument_exception") {
+					m.logger.Error(err.Error())
+					continue
```

Comment:
@batrov just trying to understand here, if we have invalid format/argument for the field, we want to ignore it and keep processing remaining assets, but if we get any other error while upserting, we will return the error, without processing remaining assets?

Reply:
yes that is expected, that way the backed up index won't get deleted and we can analyze other error before retrying the sync job.